web/src/components/button/ButtonForm/index.tsx

```diff
@@ -17,7 +17,6 @@ import FieldCustomFields from '../ButtonType/CustomFields/FieldCustomFields';
 import FieldImageUploads from 'elements/Fields/FieldImagesUpload';
 import { alertService } from 'services/Alert';
 import { CookiesState } from 'state/HomeInfo';
-import { AcceptCookiesWarn } from 'components/home/CookiesBanner';
```

Comment:
@watchoutfreedom any reason you decided to remove this? the user won't understand why can't submit